core/src/trezor/messages/MessageType.py

```diff
@@ -67,6 +67,7 @@
 DebugLinkMemory = 111
 DebugLinkMemoryWrite = 112
 DebugLinkFlashErase = 113
+DebugLinkLayout = 9001
```

Comment:
Does the 99 prefix make sense if the other Debug messages do not have it?

Reply:
Hm this is quite a mess. I'd vote to change the other DebugLink message then, but maybe it does more harm than good. Any opinions on this @prusnak?

Reply:
the problem with changing existing debuglink codes is that it will break upgrade-tests

Reply:
Ay. Hmm, let's keep it, I think it's more of an OCD thing anyway.